main.js

```diff
@@ -99,9 +100,11 @@ function createChallengeBox(challengeData) {
     btn.classList.add('main__sliderBox--button')
     btn.textContent = 'Book this room';
   }
+
   btn.classList.add('red');
+  //to show first modal
+  btn.addEventListener("click", modalSection1.bind(this, challengeData.title, challengeData.id, challengeData.minParticipants, challengeData.maxParticipants));
```

Comment:
The main purpose of the `bind` method is as a way to lock down (bind) what value the `this` variable will have inside of a function. The `modalSection` function does not use `this`, so I see no reason to use `bind()` here.

Instead you could just do:

```js
btn.addEventListener('click', () => modalSection1(challengeData.title, challengeData.id, challengeData.minParticipants, challengeData.participants));
```

Or refactor `modalSection1` to accept the challenge data in full:

```js
btn.addEventListener('click', () => modalSection1(challengeData));
```